subsys/net/lib/fota_download/src/fota_download.c

```diff
@@ -222,6 +225,7 @@ static int download_client_callback(const struct download_client_evt *event)
 			/* Return non-zero to tell download_client to stop */
 			return event->error;
 		}
+		download_started = false;
```

Comment:
Should this be moved further up in the handling of this case? There is a `return` in the `else` which would stop this from being set.

Reply:
I see now that we actually don't need this here since we do it in `send_error_evt` should we remove this then?

Reply:
yeah. I was just safe guarding a bit oo much.